ovirt_imageio/_internal/handlers/tickets.py

```diff
@@ -52,7 +52,7 @@ def put(self, req, resp, ticket_id):
                 http.BAD_REQUEST,
                 "Ticket is not in a json format: {}".format(e))
 
-        log.info("[%s] ADD ticket=%s", req.client_addr, ticket_dict)
+        log.info("[%s] ADD transfer=%s", req.client_addr, ticket_dict)
```

Comment:
ticket_dict --> ticket.transfer_id
or leav it as "ticket"?

Reply:
Yes, another wrong change - ticket_dict is not a transfer id, should be kept as ticket=